ci/cases/pr/C96C48_ufs_hybatmDA.yaml

```diff
@@ -5,18 +5,17 @@ experiment:
 arguments:
   pslot: {{ 'pslot' | getenv }}
   app: ATM
-  resdetatmos: 48
+  resdetatmos: 96
   resensatmos: 48
   comroot: {{ 'RUNTESTS' | getenv }}/COMROOT
   expdir: {{ 'RUNTESTS' | getenv }}/EXPDIR
-  idate: 2021032312
-  edate: 2021032400
+  icsdir: {{ 'ICSDIR_ROOT' | getenv }}/C96C48
+  idate: 2024022318
+  edate: 2024022400
   nens: 2
   gfs_cyc: 1
   start: warm
   yaml: {{ HOMEgfs }}/ci/cases/yamls/ufs_hybatmDA_defaults.ci.yaml
 
 skip_ci_on_hosts:
-  - hera
-  - orion
```

Comment:
I think we need to retain this till we can setup the experiment with `setup_expy.py` or update the staging mechanism to do it for us.  
It is an item on the TODO list for us, but unsure about the priority given current resources.

Reply:
Requested change committed at 80639a01.

I strongly encourage execution of JEDI ATM in g-w CI sooner rather than later.  JEDI ATM prototype cycling is now running.  We want to ensure g-w updates do not break this capability.

Earlier conversations pointed me at using `create_experiment.py` to set up JEDI ATM CI.  I modified the JEDI ATM CI case so that the staging invoked by `create_experiment.py` works as is.  The 2024022318 to 2024022400 JEDI ATM cycle reported in this PR began with `create_experiment.py` followed by rocotorun in EXPDIR.